src/bootstrap/config/tests.rs

```diff
@@ -94,3 +92,36 @@ fn detect_src_and_out() {
 fn clap_verify() {
     Flags::command().debug_assert();
 }
+
+#[test]
+fn override_toml() {
```

Comment:
Could you add some more tests, please? Lists, top-level options, integers, dictionaries (e.g. `llvm.build-config`), duplicated values would all be good to test. Looks like right now duplicated values are silently ignored - I think they should be a hard error instead, like a normal toml file.

Reply:
I've updated with some more tests covering these cases. Currently, strings don't need to be quoted as a top-level value, but do inside lists and dictionaries to make parsing simpler. I believe this is an improvement over the current behaviour of `configure.py`